browsers/oculus.json

```diff
@@ -0,0 +1,218 @@
+{
+  "browsers": {
+    "oculus": {
+      "accepts_flags": true,
+      "name": "Oculus Browser",
+      "pref_url": "chrome://flags",
```

Comment:
Just double checking this is the correct URL?

Reply:
Yep. For various reasons Oculus Browser uses the `chrome:` scheme for these surfaces; `chrome://flags` is right.